src/python/pants/base/specs_integration_test.py

```diff
@@ -154,7 +154,8 @@ def test_file_arg() -> None:
         assert run(
             ["list", f"{tmpdir}/py/app.py", f"{tmpdir}/py/utils/strutil_test.py"]
         ).stdout.splitlines() == [
-            f"{tmpdir}/py:bin",
+            # Uncomment once https://github.com/pantsbuild/pants/issues/16772 is resolved
```

Comment:
Actually this test is now stale. The pex binary is no longer a "secondary owner" of the python source.